types/reload/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for reload 3.2
+// Project: https://github.com/alallier/reload#readme
+// Definitions by: Victor Nascimento Bakke <https://github.com/Gipphe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import express = require('express');
+import ws = require('ws');
+
+declare function reload(
+    app: express.Express,
+    opts: reload.OptionsWaitToStartWebSocketServer,
```

Comment:
```suggestion
    opts?: reload.OptionsWaitToStartWebSocketServer,
```
this one seems to be optional as used in readme:
https://github.com/alallier/reload#express-example
and in source code:
https://github.com/alallier/reload/blob/e7f8234d31a20d0e5142e6959097c1c8011d58c1/lib/reload.js#L9

Reply:
It is optional, as noted in the second overload. This overload deals specifically with passing `webSocketServerWaitStart: true` in the `opts` object, which causes `reload` to return a `startWebSocketServer(): Promise<Reload>` method in the resolved object.

Reply:
I'm really sorry, I've missed overloaded func, call me lazy :)

Reply:
No worries :) Thank you for reviewing this PR, and thank you for the work you do!